internal/gke/deploy.go

```diff
@@ -849,6 +1087,21 @@ func ensureManagedCluster(ctx context.Context, config CloudConfig, name, region
 		return nil, err
 	}
 
+	// Setup the subordinate Certificate Authority in the cluster region.
+	if err := ensureSubordinateCA(ctx, config, region); err != nil {
```

Comment:
I noticed we call ensureSubordinateCA above as well. I'm not fully following the flow, but just checking we're not redundantly calling it?

Reply:
Good catch, we were!